docs/pages/reference/config.mdx

```diff
@@ -484,6 +484,23 @@ auth_service:
     # See "TLS Routing" in Architecture section for additional information.
     proxy_listener_mode: multiplex
 
+    # Determines the strategy that the cluster uses for connecting clients to
+    # agents in IoT mode.
+    tunnel_strategy:
+      # Possible tunnel strategy types are:
+      #
+      # "agent_mesh":    The default behavior, where agents will connect to every
+      #                  Teleport Proxy Service instance.
+      # "proxy_peering": Agents will connect to a subset of Proxy Service instances
+      #                  and clients will be routed between Proxy Service instances
+      #                  for end-to-end connectivity.
+      type: proxy_peering
+
+      # The number of reverse tunnel connections agents will attempt to create.
+      # This field is only available when using the "proxy_peering" tunnel
+      # strategy type.
```

Comment:
Should we mention that this should be set to >1 for high availability?

Reply:
Updated here [593d6ac](https://github.com/gravitational/teleport/pull/15790/commits/593d6aca044d9dcfe96fd28099dbcedc9e2d431e)